coderd/database/dbgen/dbgen.go

```diff
@@ -220,16 +220,28 @@ func WorkspaceAgentScriptTimings(t testing.TB, db database.Store, script databas
 }
 
 func WorkspaceAgentScriptTiming(t testing.TB, db database.Store, orig database.WorkspaceAgentScriptTiming) database.WorkspaceAgentScriptTiming {
-	timing, err := db.InsertWorkspaceAgentScriptTimings(genCtx, database.InsertWorkspaceAgentScriptTimingsParams{
-		StartedAt: takeFirst(orig.StartedAt, dbtime.Now()),
-		EndedAt:   takeFirst(orig.EndedAt, dbtime.Now()),
-		Stage:     takeFirst(orig.Stage, database.WorkspaceAgentScriptTimingStageStart),
-		ScriptID:  takeFirst(orig.ScriptID, uuid.New()),
-		ExitCode:  takeFirst(orig.ExitCode, 0),
-		Status:    takeFirst(orig.Status, database.WorkspaceAgentScriptTimingStatusOk),
-	})
-	require.NoError(t, err, "insert workspace agent script")
-	return timing
+	for i := 0; i < 2500; i++ {
```

Comment:
I'm guessing you didn't mean to add leave this loop

Reply:
2500 retries was a little excessive, lowered it to 10